uplink/src/collector/mod.rs

```diff
@@ -1,2 +1,3 @@
 pub mod simulator;
 pub mod tcpjson;
+pub mod telemetry;
```

Comment:
Since this is already in `collector` module, renaming this to `systemtelemetry` or `systemstats` might be more intuitive